src/Components/Web.JS/src/Rendering/BrowserRenderer.ts

```diff
@@ -560,6 +564,8 @@ function stripOnPrefix(attributeName: string) {
   throw new Error(`Attribute should be an event name, but doesn't start with 'on'. Value: '${attributeName}'`);
 }
 
+type BlazorHtmlSelectElement = (HTMLSelectElement & { _blazorDeferredValue?: string })
```

Comment:
```suggestion
type BlazorHtmlSelectElement = HTMLSelectElement & { _blazorDeferredValue?: string };
```

But if you're defining this type, could we also use it, and remove the references to `selectElem[deferredValuePropname]` since they could now be `selectElem._blazorDeferredValue`?